src/adapters/bidderFactory.js

```diff
@@ -307,6 +307,7 @@ export function newBidder(spec) {
               // creating a copy of original values as cpm and currency are modified later
               bid.originalCpm = bid.cpm;
               bid.originalCurrency = bid.currency;
+              bid.meta = bid.meta || Object.assign({}, bid[bidRequest.bidder]);
```

Comment:
Took me a couple minutes to realize what this was. Figured it out, then noticed you explained it in the PR text as well haha 🤦 

Looks good!